2018-07-15 09:46:27.3135|1|ERROR|Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware|An unhandled exception has occurred while executing the request. System.AggregateException: One or more errors occurred. (I'm sorry I can't parse the current expression) ---> SqlSugar.UtilExceptions: I'm sorry I can't parse the current expression
   at SqlSugar.Check.Exception(Boolean isException, String message, String[] args)
   at SqlSugar.SubResolve..ctor(MethodCallExpression expression, ExpressionContext context, Expression oppsiteExpression)
   at SqlSugar.MethodCallExpressionResolve..ctor(ExpressionParameter parameter)
   at SqlSugar.BaseResolve.Start()
   at SqlSugar.BinaryExpressionResolve..ctor(ExpressionParameter parameter)
   at SqlSugar.BaseResolve.Start()
   at SqlSugar.ExpressionContext.Resolve(Expression expression, ResolveExpressType resolveType)
   at SqlSugar.BaseResolve.ResolveNewExpressions(ExpressionParameter parameter, Expression item, String asName)
   at SqlSugar.MemberInitExpressionResolve.Select(MemberInitExpression expression, ExpressionParameter parameter, Boolean isSingle)
   at SqlSugar.MemberInitExpressionResolve..ctor(ExpressionParameter parameter)
   at SqlSugar.BaseResolve.Start()
   at SqlSugar.LambdaExpressionResolve..ctor(ExpressionParameter parameter)
   at SqlSugar.BaseResolve.Start()
   at SqlSugar.ExpressionContext.Resolve(Expression expression, ResolveExpressType resolveType)
   at SqlSugar.QueryBuilder.GetExpressionValue(Expression expression, ResolveExpressType resolveType)
   at SqlSugar.QueryBuilder.GetSelectValueByExpression()
   at SqlSugar.MySqlQueryBuilder.get_GetSelectValue()
   at SqlSugar.MySqlQueryBuilder.ToSqlString()
   at SqlSugar.QueryableProvider`1.ToSql()
   at SqlSugar.QueryableProvider`1._ToList[TResult]()
   at SqlSugar.QueryableProvider`1.ToList()
   at SqlSugar.QueryableProvider`1.First()
   at FytSoa.Service.Implements.ErpGoodsSkuService.GetByCodeAsync(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Service\Implements\Erp\ErpGoodsSkuService.cs:line 138
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at FytSoa.Api.Areas.APP.Controllers.GoodsController.GetGoodsByCode(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Api\Areas\APP\Controllers\GoodsController.cs:line 34
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Cors.Infrastructure.CorsMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
---> (Inner Exception #0) SqlSugar.UtilExceptions: I'm sorry I can't parse the current expression
   at SqlSugar.Check.Exception(Boolean isException, String message, String[] args)
   at SqlSugar.SubResolve..ctor(MethodCallExpression expression, ExpressionContext context, Expression oppsiteExpression)
   at SqlSugar.MethodCallExpressionResolve..ctor(ExpressionParameter parameter)
   at SqlSugar.BaseResolve.Start()
   at SqlSugar.BinaryExpressionResolve..ctor(ExpressionParameter parameter)
   at SqlSugar.BaseResolve.Start()
   at SqlSugar.ExpressionContext.Resolve(Expression expression, ResolveExpressType resolveType)
   at SqlSugar.BaseResolve.ResolveNewExpressions(ExpressionParameter parameter, Expression item, String asName)
   at SqlSugar.MemberInitExpressionResolve.Select(MemberInitExpression expression, ExpressionParameter parameter, Boolean isSingle)
   at SqlSugar.MemberInitExpressionResolve..ctor(ExpressionParameter parameter)
   at SqlSugar.BaseResolve.Start()
   at SqlSugar.LambdaExpressionResolve..ctor(ExpressionParameter parameter)
   at SqlSugar.BaseResolve.Start()
   at SqlSugar.ExpressionContext.Resolve(Expression expression, ResolveExpressType resolveType)
   at SqlSugar.QueryBuilder.GetExpressionValue(Expression expression, ResolveExpressType resolveType)
   at SqlSugar.QueryBuilder.GetSelectValueByExpression()
   at SqlSugar.MySqlQueryBuilder.get_GetSelectValue()
   at SqlSugar.MySqlQueryBuilder.ToSqlString()
   at SqlSugar.QueryableProvider`1.ToSql()
   at SqlSugar.QueryableProvider`1._ToList[TResult]()
   at SqlSugar.QueryableProvider`1.ToList()
   at SqlSugar.QueryableProvider`1.First()
   at FytSoa.Service.Implements.ErpGoodsSkuService.GetByCodeAsync(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Service\Implements\Erp\ErpGoodsSkuService.cs:line 138<---
|url: |action:  |IP: |QueryString: 
2018-07-15 09:50:49.9411|1|ERROR|Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware|An unhandled exception has occurred while executing the request. System.AggregateException: One or more errors occurred. (I'm sorry I can't parse the current expression) ---> SqlSugar.UtilExceptions: I'm sorry I can't parse the current expression
   at SqlSugar.Check.Exception(Boolean isException, String message, String[] args)
   at SqlSugar.SubResolve..ctor(MethodCallExpression expression, ExpressionContext context, Expression oppsiteExpression)
   at SqlSugar.MethodCallExpressionResolve..ctor(ExpressionParameter parameter)
   at SqlSugar.BaseResolve.Start()
   at SqlSugar.BinaryExpressionResolve..ctor(ExpressionParameter parameter)
   at SqlSugar.BaseResolve.Start()
   at SqlSugar.ExpressionContext.Resolve(Expression expression, ResolveExpressType resolveType)
   at SqlSugar.BaseResolve.ResolveNewExpressions(ExpressionParameter parameter, Expression item, String asName)
   at SqlSugar.MemberInitExpressionResolve.Select(MemberInitExpression expression, ExpressionParameter parameter, Boolean isSingle)
   at SqlSugar.MemberInitExpressionResolve..ctor(ExpressionParameter parameter)
   at SqlSugar.BaseResolve.Start()
   at SqlSugar.LambdaExpressionResolve..ctor(ExpressionParameter parameter)
   at SqlSugar.BaseResolve.Start()
   at SqlSugar.ExpressionContext.Resolve(Expression expression, ResolveExpressType resolveType)
   at SqlSugar.QueryBuilder.GetExpressionValue(Expression expression, ResolveExpressType resolveType)
   at SqlSugar.QueryBuilder.GetSelectValueByExpression()
   at SqlSugar.MySqlQueryBuilder.get_GetSelectValue()
   at SqlSugar.MySqlQueryBuilder.ToSqlString()
   at SqlSugar.QueryableProvider`1.ToSql()
   at SqlSugar.QueryableProvider`1._ToList[TResult]()
   at SqlSugar.QueryableProvider`1.ToList()
   at SqlSugar.QueryableProvider`1.First()
   at FytSoa.Service.Implements.ErpGoodsSkuService.GetByCodeAsync(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Service\Implements\Erp\ErpGoodsSkuService.cs:line 138
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at FytSoa.Api.Areas.APP.Controllers.GoodsController.GetGoodsByCode(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Api\Areas\APP\Controllers\GoodsController.cs:line 34
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Cors.Infrastructure.CorsMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
---> (Inner Exception #0) SqlSugar.UtilExceptions: I'm sorry I can't parse the current expression
   at SqlSugar.Check.Exception(Boolean isException, String message, String[] args)
   at SqlSugar.SubResolve..ctor(MethodCallExpression expression, ExpressionContext context, Expression oppsiteExpression)
   at SqlSugar.MethodCallExpressionResolve..ctor(ExpressionParameter parameter)
   at SqlSugar.BaseResolve.Start()
   at SqlSugar.BinaryExpressionResolve..ctor(ExpressionParameter parameter)
   at SqlSugar.BaseResolve.Start()
   at SqlSugar.ExpressionContext.Resolve(Expression expression, ResolveExpressType resolveType)
   at SqlSugar.BaseResolve.ResolveNewExpressions(ExpressionParameter parameter, Expression item, String asName)
   at SqlSugar.MemberInitExpressionResolve.Select(MemberInitExpression expression, ExpressionParameter parameter, Boolean isSingle)
   at SqlSugar.MemberInitExpressionResolve..ctor(ExpressionParameter parameter)
   at SqlSugar.BaseResolve.Start()
   at SqlSugar.LambdaExpressionResolve..ctor(ExpressionParameter parameter)
   at SqlSugar.BaseResolve.Start()
   at SqlSugar.ExpressionContext.Resolve(Expression expression, ResolveExpressType resolveType)
   at SqlSugar.QueryBuilder.GetExpressionValue(Expression expression, ResolveExpressType resolveType)
   at SqlSugar.QueryBuilder.GetSelectValueByExpression()
   at SqlSugar.MySqlQueryBuilder.get_GetSelectValue()
   at SqlSugar.MySqlQueryBuilder.ToSqlString()
   at SqlSugar.QueryableProvider`1.ToSql()
   at SqlSugar.QueryableProvider`1._ToList[TResult]()
   at SqlSugar.QueryableProvider`1.ToList()
   at SqlSugar.QueryableProvider`1.First()
   at FytSoa.Service.Implements.ErpGoodsSkuService.GetByCodeAsync(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Service\Implements\Erp\ErpGoodsSkuService.cs:line 138<---
|url: |action:  |IP: |QueryString: 
2018-07-15 09:51:13.1067|1|ERROR|Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware|An unhandled exception has occurred while executing the request. System.AggregateException: One or more errors occurred. (I'm sorry I can't parse the current expression) ---> SqlSugar.UtilExceptions: I'm sorry I can't parse the current expression
   at SqlSugar.Check.Exception(Boolean isException, String message, String[] args)
   at SqlSugar.SubResolve..ctor(MethodCallExpression expression, ExpressionContext context, Expression oppsiteExpression)
   at SqlSugar.MethodCallExpressionResolve..ctor(ExpressionParameter parameter)
   at SqlSugar.BaseResolve.Start()
   at SqlSugar.BinaryExpressionResolve..ctor(ExpressionParameter parameter)
   at SqlSugar.BaseResolve.Start()
   at SqlSugar.ExpressionContext.Resolve(Expression expression, ResolveExpressType resolveType)
   at SqlSugar.BaseResolve.ResolveNewExpressions(ExpressionParameter parameter, Expression item, String asName)
   at SqlSugar.MemberInitExpressionResolve.Select(MemberInitExpression expression, ExpressionParameter parameter, Boolean isSingle)
   at SqlSugar.MemberInitExpressionResolve..ctor(ExpressionParameter parameter)
   at SqlSugar.BaseResolve.Start()
   at SqlSugar.LambdaExpressionResolve..ctor(ExpressionParameter parameter)
   at SqlSugar.BaseResolve.Start()
   at SqlSugar.ExpressionContext.Resolve(Expression expression, ResolveExpressType resolveType)
   at SqlSugar.QueryBuilder.GetExpressionValue(Expression expression, ResolveExpressType resolveType)
   at SqlSugar.QueryBuilder.GetSelectValueByExpression()
   at SqlSugar.MySqlQueryBuilder.get_GetSelectValue()
   at SqlSugar.MySqlQueryBuilder.ToSqlString()
   at SqlSugar.QueryableProvider`1.ToSql()
   at SqlSugar.QueryableProvider`1._ToList[TResult]()
   at SqlSugar.QueryableProvider`1.ToList()
   at SqlSugar.QueryableProvider`1.First()
   at FytSoa.Service.Implements.ErpGoodsSkuService.GetByCodeAsync(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Service\Implements\Erp\ErpGoodsSkuService.cs:line 138
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at FytSoa.Api.Areas.APP.Controllers.GoodsController.GetGoodsByCode(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Api\Areas\APP\Controllers\GoodsController.cs:line 34
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Cors.Infrastructure.CorsMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
---> (Inner Exception #0) SqlSugar.UtilExceptions: I'm sorry I can't parse the current expression
   at SqlSugar.Check.Exception(Boolean isException, String message, String[] args)
   at SqlSugar.SubResolve..ctor(MethodCallExpression expression, ExpressionContext context, Expression oppsiteExpression)
   at SqlSugar.MethodCallExpressionResolve..ctor(ExpressionParameter parameter)
   at SqlSugar.BaseResolve.Start()
   at SqlSugar.BinaryExpressionResolve..ctor(ExpressionParameter parameter)
   at SqlSugar.BaseResolve.Start()
   at SqlSugar.ExpressionContext.Resolve(Expression expression, ResolveExpressType resolveType)
   at SqlSugar.BaseResolve.ResolveNewExpressions(ExpressionParameter parameter, Expression item, String asName)
   at SqlSugar.MemberInitExpressionResolve.Select(MemberInitExpression expression, ExpressionParameter parameter, Boolean isSingle)
   at SqlSugar.MemberInitExpressionResolve..ctor(ExpressionParameter parameter)
   at SqlSugar.BaseResolve.Start()
   at SqlSugar.LambdaExpressionResolve..ctor(ExpressionParameter parameter)
   at SqlSugar.BaseResolve.Start()
   at SqlSugar.ExpressionContext.Resolve(Expression expression, ResolveExpressType resolveType)
   at SqlSugar.QueryBuilder.GetExpressionValue(Expression expression, ResolveExpressType resolveType)
   at SqlSugar.QueryBuilder.GetSelectValueByExpression()
   at SqlSugar.MySqlQueryBuilder.get_GetSelectValue()
   at SqlSugar.MySqlQueryBuilder.ToSqlString()
   at SqlSugar.QueryableProvider`1.ToSql()
   at SqlSugar.QueryableProvider`1._ToList[TResult]()
   at SqlSugar.QueryableProvider`1.ToList()
   at SqlSugar.QueryableProvider`1.First()
   at FytSoa.Service.Implements.ErpGoodsSkuService.GetByCodeAsync(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Service\Implements\Erp\ErpGoodsSkuService.cs:line 138<---
|url: |action:  |IP: |QueryString: 
2018-07-15 09:51:37.7171|1|ERROR|Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware|An unhandled exception has occurred while executing the request. System.AggregateException: One or more errors occurred. (Unknown column 'm.Counts' in 'field list') ---> MySql.Data.MySqlClient.MySqlException: Unknown column 'm.Counts' in 'field list'
   at SqlSugar.AdoProvider.GetDataReader(String sql, SugarParameter[] parameters)
   at SqlSugar.QueryableProvider`1.GetData[TResult](KeyValuePair`2 sqlObj)
   at SqlSugar.QueryableProvider`1._ToList[TResult]()
   at SqlSugar.QueryableProvider`1.Sum[TResult](String sumField)
   at SqlSugar.QueryableProvider`1._Sum[TResult](Expression expression)
   at SqlSugar.QueryableProvider`1.Sum[TResult](Expression`1 expression)
   at FytSoa.Service.Implements.ErpGoodsSkuService.GetByCodeAsync(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Service\Implements\Erp\ErpGoodsSkuService.cs:line 152
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at FytSoa.Api.Areas.APP.Controllers.GoodsController.GetGoodsByCode(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Api\Areas\APP\Controllers\GoodsController.cs:line 34
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Cors.Infrastructure.CorsMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
---> (Inner Exception #0) MySql.Data.MySqlClient.MySqlException (0x80004005): Unknown column 'm.Counts' in 'field list'
   at SqlSugar.AdoProvider.GetDataReader(String sql, SugarParameter[] parameters)
   at SqlSugar.QueryableProvider`1.GetData[TResult](KeyValuePair`2 sqlObj)
   at SqlSugar.QueryableProvider`1._ToList[TResult]()
   at SqlSugar.QueryableProvider`1.Sum[TResult](String sumField)
   at SqlSugar.QueryableProvider`1._Sum[TResult](Expression expression)
   at SqlSugar.QueryableProvider`1.Sum[TResult](Expression`1 expression)
   at FytSoa.Service.Implements.ErpGoodsSkuService.GetByCodeAsync(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Service\Implements\Erp\ErpGoodsSkuService.cs:line 152<---
|url: |action:  |IP: |QueryString: 
2018-07-15 09:51:40.0563|1|ERROR|Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware|An unhandled exception has occurred while executing the request. System.AggregateException: One or more errors occurred. (Unknown column 'm.Counts' in 'field list') ---> MySql.Data.MySqlClient.MySqlException: Unknown column 'm.Counts' in 'field list'
   at SqlSugar.AdoProvider.GetDataReader(String sql, SugarParameter[] parameters)
   at SqlSugar.QueryableProvider`1.GetData[TResult](KeyValuePair`2 sqlObj)
   at SqlSugar.QueryableProvider`1._ToList[TResult]()
   at SqlSugar.QueryableProvider`1.Sum[TResult](String sumField)
   at SqlSugar.QueryableProvider`1._Sum[TResult](Expression expression)
   at SqlSugar.QueryableProvider`1.Sum[TResult](Expression`1 expression)
   at FytSoa.Service.Implements.ErpGoodsSkuService.GetByCodeAsync(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Service\Implements\Erp\ErpGoodsSkuService.cs:line 152
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at FytSoa.Api.Areas.APP.Controllers.GoodsController.GetGoodsByCode(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Api\Areas\APP\Controllers\GoodsController.cs:line 34
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Cors.Infrastructure.CorsMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
---> (Inner Exception #0) MySql.Data.MySqlClient.MySqlException (0x80004005): Unknown column 'm.Counts' in 'field list'
   at SqlSugar.AdoProvider.GetDataReader(String sql, SugarParameter[] parameters)
   at SqlSugar.QueryableProvider`1.GetData[TResult](KeyValuePair`2 sqlObj)
   at SqlSugar.QueryableProvider`1._ToList[TResult]()
   at SqlSugar.QueryableProvider`1.Sum[TResult](String sumField)
   at SqlSugar.QueryableProvider`1._Sum[TResult](Expression expression)
   at SqlSugar.QueryableProvider`1.Sum[TResult](Expression`1 expression)
   at FytSoa.Service.Implements.ErpGoodsSkuService.GetByCodeAsync(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Service\Implements\Erp\ErpGoodsSkuService.cs:line 152<---
|url: |action:  |IP: |QueryString: 
2018-07-15 09:51:40.5705|1|ERROR|Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware|An unhandled exception has occurred while executing the request. System.AggregateException: One or more errors occurred. (Unknown column 'm.Counts' in 'field list') ---> MySql.Data.MySqlClient.MySqlException: Unknown column 'm.Counts' in 'field list'
   at SqlSugar.AdoProvider.GetDataReader(String sql, SugarParameter[] parameters)
   at SqlSugar.QueryableProvider`1.GetData[TResult](KeyValuePair`2 sqlObj)
   at SqlSugar.QueryableProvider`1._ToList[TResult]()
   at SqlSugar.QueryableProvider`1.Sum[TResult](String sumField)
   at SqlSugar.QueryableProvider`1._Sum[TResult](Expression expression)
   at SqlSugar.QueryableProvider`1.Sum[TResult](Expression`1 expression)
   at FytSoa.Service.Implements.ErpGoodsSkuService.GetByCodeAsync(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Service\Implements\Erp\ErpGoodsSkuService.cs:line 152
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at FytSoa.Api.Areas.APP.Controllers.GoodsController.GetGoodsByCode(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Api\Areas\APP\Controllers\GoodsController.cs:line 34
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Cors.Infrastructure.CorsMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
---> (Inner Exception #0) MySql.Data.MySqlClient.MySqlException (0x80004005): Unknown column 'm.Counts' in 'field list'
   at SqlSugar.AdoProvider.GetDataReader(String sql, SugarParameter[] parameters)
   at SqlSugar.QueryableProvider`1.GetData[TResult](KeyValuePair`2 sqlObj)
   at SqlSugar.QueryableProvider`1._ToList[TResult]()
   at SqlSugar.QueryableProvider`1.Sum[TResult](String sumField)
   at SqlSugar.QueryableProvider`1._Sum[TResult](Expression expression)
   at SqlSugar.QueryableProvider`1.Sum[TResult](Expression`1 expression)
   at FytSoa.Service.Implements.ErpGoodsSkuService.GetByCodeAsync(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Service\Implements\Erp\ErpGoodsSkuService.cs:line 152<---
|url: |action:  |IP: |QueryString: 
2018-07-15 09:51:40.9233|1|ERROR|Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware|An unhandled exception has occurred while executing the request. System.AggregateException: One or more errors occurred. (Unknown column 'm.Counts' in 'field list') ---> MySql.Data.MySqlClient.MySqlException: Unknown column 'm.Counts' in 'field list'
   at SqlSugar.AdoProvider.GetDataReader(String sql, SugarParameter[] parameters)
   at SqlSugar.QueryableProvider`1.GetData[TResult](KeyValuePair`2 sqlObj)
   at SqlSugar.QueryableProvider`1._ToList[TResult]()
   at SqlSugar.QueryableProvider`1.Sum[TResult](String sumField)
   at SqlSugar.QueryableProvider`1._Sum[TResult](Expression expression)
   at SqlSugar.QueryableProvider`1.Sum[TResult](Expression`1 expression)
   at FytSoa.Service.Implements.ErpGoodsSkuService.GetByCodeAsync(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Service\Implements\Erp\ErpGoodsSkuService.cs:line 152
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at FytSoa.Api.Areas.APP.Controllers.GoodsController.GetGoodsByCode(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Api\Areas\APP\Controllers\GoodsController.cs:line 34
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Cors.Infrastructure.CorsMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
---> (Inner Exception #0) MySql.Data.MySqlClient.MySqlException (0x80004005): Unknown column 'm.Counts' in 'field list'
   at SqlSugar.AdoProvider.GetDataReader(String sql, SugarParameter[] parameters)
   at SqlSugar.QueryableProvider`1.GetData[TResult](KeyValuePair`2 sqlObj)
   at SqlSugar.QueryableProvider`1._ToList[TResult]()
   at SqlSugar.QueryableProvider`1.Sum[TResult](String sumField)
   at SqlSugar.QueryableProvider`1._Sum[TResult](Expression expression)
   at SqlSugar.QueryableProvider`1.Sum[TResult](Expression`1 expression)
   at FytSoa.Service.Implements.ErpGoodsSkuService.GetByCodeAsync(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Service\Implements\Erp\ErpGoodsSkuService.cs:line 152<---
|url: |action:  |IP: |QueryString: 
2018-07-15 09:51:41.2872|1|ERROR|Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware|An unhandled exception has occurred while executing the request. System.AggregateException: One or more errors occurred. (Unknown column 'm.Counts' in 'field list') ---> MySql.Data.MySqlClient.MySqlException: Unknown column 'm.Counts' in 'field list'
   at SqlSugar.AdoProvider.GetDataReader(String sql, SugarParameter[] parameters)
   at SqlSugar.QueryableProvider`1.GetData[TResult](KeyValuePair`2 sqlObj)
   at SqlSugar.QueryableProvider`1._ToList[TResult]()
   at SqlSugar.QueryableProvider`1.Sum[TResult](String sumField)
   at SqlSugar.QueryableProvider`1._Sum[TResult](Expression expression)
   at SqlSugar.QueryableProvider`1.Sum[TResult](Expression`1 expression)
   at FytSoa.Service.Implements.ErpGoodsSkuService.GetByCodeAsync(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Service\Implements\Erp\ErpGoodsSkuService.cs:line 152
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at FytSoa.Api.Areas.APP.Controllers.GoodsController.GetGoodsByCode(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Api\Areas\APP\Controllers\GoodsController.cs:line 34
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Cors.Infrastructure.CorsMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
---> (Inner Exception #0) MySql.Data.MySqlClient.MySqlException (0x80004005): Unknown column 'm.Counts' in 'field list'
   at SqlSugar.AdoProvider.GetDataReader(String sql, SugarParameter[] parameters)
   at SqlSugar.QueryableProvider`1.GetData[TResult](KeyValuePair`2 sqlObj)
   at SqlSugar.QueryableProvider`1._ToList[TResult]()
   at SqlSugar.QueryableProvider`1.Sum[TResult](String sumField)
   at SqlSugar.QueryableProvider`1._Sum[TResult](Expression expression)
   at SqlSugar.QueryableProvider`1.Sum[TResult](Expression`1 expression)
   at FytSoa.Service.Implements.ErpGoodsSkuService.GetByCodeAsync(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Service\Implements\Erp\ErpGoodsSkuService.cs:line 152<---
|url: |action:  |IP: |QueryString: 
2018-07-15 09:51:41.6018|1|ERROR|Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware|An unhandled exception has occurred while executing the request. System.AggregateException: One or more errors occurred. (Unknown column 'm.Counts' in 'field list') ---> MySql.Data.MySqlClient.MySqlException: Unknown column 'm.Counts' in 'field list'
   at SqlSugar.AdoProvider.GetDataReader(String sql, SugarParameter[] parameters)
   at SqlSugar.QueryableProvider`1.GetData[TResult](KeyValuePair`2 sqlObj)
   at SqlSugar.QueryableProvider`1._ToList[TResult]()
   at SqlSugar.QueryableProvider`1.Sum[TResult](String sumField)
   at SqlSugar.QueryableProvider`1._Sum[TResult](Expression expression)
   at SqlSugar.QueryableProvider`1.Sum[TResult](Expression`1 expression)
   at FytSoa.Service.Implements.ErpGoodsSkuService.GetByCodeAsync(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Service\Implements\Erp\ErpGoodsSkuService.cs:line 152
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at FytSoa.Api.Areas.APP.Controllers.GoodsController.GetGoodsByCode(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Api\Areas\APP\Controllers\GoodsController.cs:line 34
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Cors.Infrastructure.CorsMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
---> (Inner Exception #0) MySql.Data.MySqlClient.MySqlException (0x80004005): Unknown column 'm.Counts' in 'field list'
   at SqlSugar.AdoProvider.GetDataReader(String sql, SugarParameter[] parameters)
   at SqlSugar.QueryableProvider`1.GetData[TResult](KeyValuePair`2 sqlObj)
   at SqlSugar.QueryableProvider`1._ToList[TResult]()
   at SqlSugar.QueryableProvider`1.Sum[TResult](String sumField)
   at SqlSugar.QueryableProvider`1._Sum[TResult](Expression expression)
   at SqlSugar.QueryableProvider`1.Sum[TResult](Expression`1 expression)
   at FytSoa.Service.Implements.ErpGoodsSkuService.GetByCodeAsync(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Service\Implements\Erp\ErpGoodsSkuService.cs:line 152<---
|url: |action:  |IP: |QueryString: 
2018-07-15 09:51:41.9142|1|ERROR|Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware|An unhandled exception has occurred while executing the request. System.AggregateException: One or more errors occurred. (Unknown column 'm.Counts' in 'field list') ---> MySql.Data.MySqlClient.MySqlException: Unknown column 'm.Counts' in 'field list'
   at SqlSugar.AdoProvider.GetDataReader(String sql, SugarParameter[] parameters)
   at SqlSugar.QueryableProvider`1.GetData[TResult](KeyValuePair`2 sqlObj)
   at SqlSugar.QueryableProvider`1._ToList[TResult]()
   at SqlSugar.QueryableProvider`1.Sum[TResult](String sumField)
   at SqlSugar.QueryableProvider`1._Sum[TResult](Expression expression)
   at SqlSugar.QueryableProvider`1.Sum[TResult](Expression`1 expression)
   at FytSoa.Service.Implements.ErpGoodsSkuService.GetByCodeAsync(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Service\Implements\Erp\ErpGoodsSkuService.cs:line 152
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at FytSoa.Api.Areas.APP.Controllers.GoodsController.GetGoodsByCode(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Api\Areas\APP\Controllers\GoodsController.cs:line 34
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Cors.Infrastructure.CorsMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
---> (Inner Exception #0) MySql.Data.MySqlClient.MySqlException (0x80004005): Unknown column 'm.Counts' in 'field list'
   at SqlSugar.AdoProvider.GetDataReader(String sql, SugarParameter[] parameters)
   at SqlSugar.QueryableProvider`1.GetData[TResult](KeyValuePair`2 sqlObj)
   at SqlSugar.QueryableProvider`1._ToList[TResult]()
   at SqlSugar.QueryableProvider`1.Sum[TResult](String sumField)
   at SqlSugar.QueryableProvider`1._Sum[TResult](Expression expression)
   at SqlSugar.QueryableProvider`1.Sum[TResult](Expression`1 expression)
   at FytSoa.Service.Implements.ErpGoodsSkuService.GetByCodeAsync(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Service\Implements\Erp\ErpGoodsSkuService.cs:line 152<---
|url: |action:  |IP: |QueryString: 
2018-07-15 09:51:42.4492|1|ERROR|Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware|An unhandled exception has occurred while executing the request. System.AggregateException: One or more errors occurred. (Unknown column 'm.Counts' in 'field list') ---> MySql.Data.MySqlClient.MySqlException: Unknown column 'm.Counts' in 'field list'
   at SqlSugar.AdoProvider.GetDataReader(String sql, SugarParameter[] parameters)
   at SqlSugar.QueryableProvider`1.GetData[TResult](KeyValuePair`2 sqlObj)
   at SqlSugar.QueryableProvider`1._ToList[TResult]()
   at SqlSugar.QueryableProvider`1.Sum[TResult](String sumField)
   at SqlSugar.QueryableProvider`1._Sum[TResult](Expression expression)
   at SqlSugar.QueryableProvider`1.Sum[TResult](Expression`1 expression)
   at FytSoa.Service.Implements.ErpGoodsSkuService.GetByCodeAsync(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Service\Implements\Erp\ErpGoodsSkuService.cs:line 152
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at FytSoa.Api.Areas.APP.Controllers.GoodsController.GetGoodsByCode(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Api\Areas\APP\Controllers\GoodsController.cs:line 34
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Cors.Infrastructure.CorsMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
---> (Inner Exception #0) MySql.Data.MySqlClient.MySqlException (0x80004005): Unknown column 'm.Counts' in 'field list'
   at SqlSugar.AdoProvider.GetDataReader(String sql, SugarParameter[] parameters)
   at SqlSugar.QueryableProvider`1.GetData[TResult](KeyValuePair`2 sqlObj)
   at SqlSugar.QueryableProvider`1._ToList[TResult]()
   at SqlSugar.QueryableProvider`1.Sum[TResult](String sumField)
   at SqlSugar.QueryableProvider`1._Sum[TResult](Expression expression)
   at SqlSugar.QueryableProvider`1.Sum[TResult](Expression`1 expression)
   at FytSoa.Service.Implements.ErpGoodsSkuService.GetByCodeAsync(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Service\Implements\Erp\ErpGoodsSkuService.cs:line 152<---
|url: |action:  |IP: |QueryString: 
2018-07-15 09:51:42.7454|1|ERROR|Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware|An unhandled exception has occurred while executing the request. System.AggregateException: One or more errors occurred. (Unknown column 'm.Counts' in 'field list') ---> MySql.Data.MySqlClient.MySqlException: Unknown column 'm.Counts' in 'field list'
   at SqlSugar.AdoProvider.GetDataReader(String sql, SugarParameter[] parameters)
   at SqlSugar.QueryableProvider`1.GetData[TResult](KeyValuePair`2 sqlObj)
   at SqlSugar.QueryableProvider`1._ToList[TResult]()
   at SqlSugar.QueryableProvider`1.Sum[TResult](String sumField)
   at SqlSugar.QueryableProvider`1._Sum[TResult](Expression expression)
   at SqlSugar.QueryableProvider`1.Sum[TResult](Expression`1 expression)
   at FytSoa.Service.Implements.ErpGoodsSkuService.GetByCodeAsync(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Service\Implements\Erp\ErpGoodsSkuService.cs:line 152
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at FytSoa.Api.Areas.APP.Controllers.GoodsController.GetGoodsByCode(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Api\Areas\APP\Controllers\GoodsController.cs:line 34
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Cors.Infrastructure.CorsMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
---> (Inner Exception #0) MySql.Data.MySqlClient.MySqlException (0x80004005): Unknown column 'm.Counts' in 'field list'
   at SqlSugar.AdoProvider.GetDataReader(String sql, SugarParameter[] parameters)
   at SqlSugar.QueryableProvider`1.GetData[TResult](KeyValuePair`2 sqlObj)
   at SqlSugar.QueryableProvider`1._ToList[TResult]()
   at SqlSugar.QueryableProvider`1.Sum[TResult](String sumField)
   at SqlSugar.QueryableProvider`1._Sum[TResult](Expression expression)
   at SqlSugar.QueryableProvider`1.Sum[TResult](Expression`1 expression)
   at FytSoa.Service.Implements.ErpGoodsSkuService.GetByCodeAsync(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Service\Implements\Erp\ErpGoodsSkuService.cs:line 152<---
|url: |action:  |IP: |QueryString: 
2018-07-15 09:51:43.0299|1|ERROR|Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware|An unhandled exception has occurred while executing the request. System.AggregateException: One or more errors occurred. (Unknown column 'm.Counts' in 'field list') ---> MySql.Data.MySqlClient.MySqlException: Unknown column 'm.Counts' in 'field list'
   at SqlSugar.AdoProvider.GetDataReader(String sql, SugarParameter[] parameters)
   at SqlSugar.QueryableProvider`1.GetData[TResult](KeyValuePair`2 sqlObj)
   at SqlSugar.QueryableProvider`1._ToList[TResult]()
   at SqlSugar.QueryableProvider`1.Sum[TResult](String sumField)
   at SqlSugar.QueryableProvider`1._Sum[TResult](Expression expression)
   at SqlSugar.QueryableProvider`1.Sum[TResult](Expression`1 expression)
   at FytSoa.Service.Implements.ErpGoodsSkuService.GetByCodeAsync(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Service\Implements\Erp\ErpGoodsSkuService.cs:line 152
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at FytSoa.Api.Areas.APP.Controllers.GoodsController.GetGoodsByCode(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Api\Areas\APP\Controllers\GoodsController.cs:line 34
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Cors.Infrastructure.CorsMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
---> (Inner Exception #0) MySql.Data.MySqlClient.MySqlException (0x80004005): Unknown column 'm.Counts' in 'field list'
   at SqlSugar.AdoProvider.GetDataReader(String sql, SugarParameter[] parameters)
   at SqlSugar.QueryableProvider`1.GetData[TResult](KeyValuePair`2 sqlObj)
   at SqlSugar.QueryableProvider`1._ToList[TResult]()
   at SqlSugar.QueryableProvider`1.Sum[TResult](String sumField)
   at SqlSugar.QueryableProvider`1._Sum[TResult](Expression expression)
   at SqlSugar.QueryableProvider`1.Sum[TResult](Expression`1 expression)
   at FytSoa.Service.Implements.ErpGoodsSkuService.GetByCodeAsync(String shopGuid, String code) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Service\Implements\Erp\ErpGoodsSkuService.cs:line 152<---
|url: |action:  |IP: |QueryString: 
2018-07-15 12:20:10.9977|1|ERROR|Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware|An unhandled exception has occurred while executing the request. System.NullReferenceException: Object reference not set to an instance of an object.
   at System.Collections.Generic.Dictionary`2.TryInsert(TKey key, TValue value, InsertionBehavior behavior)
   at System.Collections.Generic.Dictionary`2.Add(TKey key, TValue value)
   at FytSoa.Common.EnumExtension.GetEnumText(Enum en) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Common\ClientApi\EnumExtension.cs:line 52
   at FytSoa.Service.Implements.ErpBackGoodsService.GetPagesAsync(PageParm parm) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Service\Implements\Erp\ErpBackGoodsService.cs:line 153
   at FytSoa.Api.Controllers.StockController.GetBackPages(PageParm parm) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Api\Controllers\StockController.cs:line 235
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Cors.Infrastructure.CorsMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)|url: |action:  |IP: |QueryString: 
2018-07-15 12:20:10.9977|1|ERROR|Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware|An unhandled exception has occurred while executing the request. System.NullReferenceException: Object reference not set to an instance of an object.
   at System.Collections.Generic.Dictionary`2.TryInsert(TKey key, TValue value, InsertionBehavior behavior)
   at System.Collections.Generic.Dictionary`2.Add(TKey key, TValue value)
   at FytSoa.Common.EnumExtension.GetEnumText(Enum en) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Common\ClientApi\EnumExtension.cs:line 52
   at FytSoa.Service.Implements.ErpBackGoodsService.GetPagesAsync(PageParm parm) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Service\Implements\Erp\ErpBackGoodsService.cs:line 153
   at FytSoa.Api.Controllers.StockController.GetBackPages(PageParm parm) in E:\2018Project\ERP项目\SoaProJect\FytSoa.Api\Controllers\StockController.cs:line 235
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Cors.Infrastructure.CorsMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)|url: |action:  |IP: |QueryString: 
